extension.json

```diff
@@ -23,8 +23,9 @@
 			"description": "Enables reading files from disk"
 		},
 		"DFRawPath": {
-			"value": "",
-			"description": "Absolute path to raws directory, when empty use 'raws' directory in extension catalog"
+			"value": "raws",
+			"path": true,
+			"description": "Relative path to raws directory"
```

Comment:
Couldn't it also be absolute? I don't see any checks for the path root in DFRawFunctions.body.php

Reply:
When  "path": true mediawiki insert extension directory to path without any other checks.
So only relative.

https://github.com/wikimedia/mediawiki/blob/9afab251cac5b02564a68c13f25418c522df9467/includes/registration/ExtensionProcessor.php#L831